2016/05/10 23:09:39 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\protected\controllers\SiteController.php(32): CController->render('index')
#16 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php
---
2016/05/10 23:15:06 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\protected\controllers\SiteController.php(32): CController->render('index')
#16 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): SiteController->actionIndex()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php
---
2016/05/10 23:16:04 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:16:43 [error] [exception.CException] exception 'CException' with message 'Alias "bootstrap.components.Bootstrap" is invalid. Make sure it points to an existing directory or file.' in C:\xampp\htdocs\web\framework\YiiBase.php:348
Stack trace:
#0 C:\xampp\htdocs\web\framework\YiiBase.php(196): YiiBase::import('bootstrap.compo...', true)
#1 C:\xampp\htdocs\web\framework\base\CModule.php(393): YiiBase::createComponent(Array)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('bootstrap')
#3 C:\xampp\htdocs\web\protected\views\layouts\main.php(19): CModule->__get('bootstrap')
#4 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#5 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#6 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(76): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#7 C:\xampp\htdocs\web\framework\web\widgets\CContentDecorator.php(54): CContentDecorator->decorate('<div id="conten...')
#8 C:\xampp\htdocs\web\framework\web\widgets\COutputProcessor.php(44): CContentDecorator->processOutput('<div id="conten...')
#9 C:\xampp\htdocs\web\framework\web\CBaseController.php(214): COutputProcessor->run()
#10 C:\xampp\htdocs\web\framework\web\CBaseController.php(308): CBaseController->endWidget('CContentDecorat...')
#11 C:\xampp\htdocs\web\protected\views\layouts\column1.php(6): CBaseController->endContent()
#12 C:\xampp\htdocs\web\framework\web\CBaseController.php(126): require('C:\\xampp\\htdocs...')
#13 C:\xampp\htdocs\web\framework\web\CBaseController.php(95): CBaseController->renderInternal('C:\\xampp\\htdocs...', Array, true)
#14 C:\xampp\htdocs\web\framework\web\CController.php(784): CBaseController->renderFile('C:\\xampp\\htdocs...', Array, true)
#15 C:\xampp\htdocs\web\framework\web\actions\CViewAction.php(140): CController->render('pages/about')
#16 C:\xampp\htdocs\web\framework\web\actions\CAction.php(76): CViewAction->run()
#17 C:\xampp\htdocs\web\framework\web\CController.php(308): CAction->runWithParams(Array)
#18 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CViewAction))
#19 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CViewAction), Array)
#20 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('page')
#21 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('site/page')
#22 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#23 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#24 {main}
REQUEST_URI=/web/index.php?r=site/page&view=about
HTTP_REFERER=http://localhost/web/index.php?r=site/contact
---
2016/05/10 23:31:11 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:31:14 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\crud\CrudCode.php(44): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): CrudCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/crud/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/crud/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:31:16 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:32:15 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:39:00 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
2016/05/10 23:39:07 [error] [exception.CHttpException.404] exception 'CHttpException' with message 'Unable to resolve the request "gi".' in C:\xampp\htdocs\web\framework\web\CWebApplication.php:286
Stack trace:
#0 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gi')
#1 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#2 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#3 {main}
REQUEST_URI=/web/index.php?r=gi
---
2016/05/10 23:39:10 [error] [exception.CDbException] exception 'CDbException' with message 'CDbConnection failed to open the DB connection: SQLSTATE[HY000] [1045] Access denied for user 'web'@'localhost' (using password: YES)' in C:\xampp\htdocs\web\framework\db\CDbConnection.php:399
Stack trace:
#0 C:\xampp\htdocs\web\framework\db\CDbConnection.php(347): CDbConnection->open()
#1 C:\xampp\htdocs\web\framework\db\CDbConnection.php(325): CDbConnection->setActive(true)
#2 C:\xampp\htdocs\web\framework\base\CModule.php(394): CDbConnection->init()
#3 C:\xampp\htdocs\web\framework\base\CModule.php(103): CModule->getComponent('db')
#4 C:\xampp\htdocs\web\framework\gii\generators\model\ModelCode.php(60): CModule->__get('db')
#5 C:\xampp\htdocs\web\framework\web\CFormModel.php(39): ModelCode->init()
#6 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(153): CFormModel->__construct()
#7 C:\xampp\htdocs\web\framework\gii\CCodeGenerator.php(63): CCodeGenerator->prepare()
#8 C:\xampp\htdocs\web\framework\web\actions\CInlineAction.php(49): CCodeGenerator->actionIndex()
#9 C:\xampp\htdocs\web\framework\web\CController.php(308): CInlineAction->runWithParams(Array)
#10 C:\xampp\htdocs\web\framework\web\CController.php(286): CController->runAction(Object(CInlineAction))
#11 C:\xampp\htdocs\web\framework\web\CController.php(265): CController->runActionWithFilters(Object(CInlineAction), Array)
#12 C:\xampp\htdocs\web\framework\web\CWebApplication.php(282): CController->run('index')
#13 C:\xampp\htdocs\web\framework\web\CWebApplication.php(141): CWebApplication->runController('gii/model/index')
#14 C:\xampp\htdocs\web\framework\base\CApplication.php(185): CWebApplication->processRequest()
#15 C:\xampp\htdocs\web\index.php(13): CApplication->run()
#16 {main}
REQUEST_URI=/web/index.php?r=gii/model/index
HTTP_REFERER=http://localhost/web/index.php?r=gii
---
